Emergence/Contexts/Networking/ArtsyAPI.swift

```diff
@@ -0,0 +1,93 @@
+import Foundation
+import RxSwift
+import Moya
+import Alamofire
+
+enum ArtsyAPI {
+    case XApp
+    case ShowInfo(showID:String)
+
+}
+
+extension ArtsyAPI : MoyaTarget {
+
+    var base: String { return AppSetup.sharedState.useStaging ? "https://stagingapi.artsy.net" : "https://api.artsy.net" }
+    var baseURL: NSURL { return NSURL(string: base)! }
+
+    var parameters: [String: AnyObject] {
+        switch self {
+
+        case .XApp:
+            return [
+                "grant_type": "credentials"
+            ]
+
+        default:
+            return [:]
+        }
+    }
+
+    var method: Moya.Method {
+        switch self {
+        default:
+            return .GET
+        }
+    }
+
+    // Only kinda guilty. Deal with it.
```

Comment:
Whomp whomp. 

May be worth considering this as a broader case for Moya. Maybe have this as a default implementation of `sampleData` in a protocol extension?